pylintrc

```diff
@@ -96,11 +96,11 @@ class-rgx=[A-Z_][a-zA-Z0-9]+$
 
 # Regular expression which should only match correct function names
 # Amplify: Up to 40 characters long
-function-rgx=[a-z_][a-z0-9_]{2,40}$
+function-rgx=[a-z_][a-z0-9_]+$
```

Comment:
Why?

Reply:
IMHO, test function names should be as descriptive as possible while still being under the project line limit.  That way you can name your functions `test_returns_job_obj_when_something_else_is_true` and have other developers know clearly what is being tested.  

Of course, you want to assess each function name separately. You usually don't want a `@property` name being over 40 characters, but that would be something that is reviewed in the PR as well.

Reply:
Fair enough!